tools/participation-cli/main.go

```diff
@@ -67,52 +73,38 @@ func buildTransactionPayload(ctx context.Context, client *iotago.NodeHTTPAPIClie
 			return nil, err
 		}
 
-		if unspentOutput.Type() != iotago.OutputSigLockedSingleOutput {
+		if unspentOutput.Type() != iotago.OutputExtended {
 			continue
 		}
 
-		balance, err := unspentOutput.Deposit()
-		if err != nil {
-			return nil, err
-		}
-
-		inputsBalance += int(balance)
+		inputsBalance += unspentOutput.Deposit()
 		txBuilder.AddInput(&iotago.ToBeSignedUTXOInput{Address: inputAddress, Input: input})
-
-		if inputsBalance >= (int(outputAmount) + MINIMUM_DUST_BALANCE) {
```

Comment:
It's wrong to remove this check, since you will always collect all UTXOs.
It should be this instead:
```diff
-		if inputsBalance >= (int(outputAmount) + MINIMUM_DUST_BALANCE) {
+		if inputsBalance >= int(outputAmount) {
```